src/testRunner/unittests/tsbuild/helpers.ts

```diff
@@ -503,51 +499,34 @@ interface Symbol {
         modifyFs: (fs: vfs.FileSystem) => void;
         subScenario: string;
         commandLineArgs?: readonly string[];
-        cleanBuildDiscrepancies?: () => ESMap<string, CleanBuildDescrepancy>;
+        descripencyExplaination?: () => readonly string[];

```

Comment:
Why an array, rather than a single string?

Reply:
And why wrap in a function?  Is the string expensive to compute?

Reply:
These are inputs when the testcase is created so this will ensure calculation only if needed

Reply:
Array just helps keep lines even as we are anyways joining the strings letter

Reply:
The calculation we're postponing is an array of string literals?  If you're going to keep the array, please add a comment about how to interpret the elements.